packages/mobile/src/screens/coin-details-screen/components/coinMetricsUtil.ts

```diff
@@ -0,0 +1,87 @@
+import { formatCount, formatCurrencyWithMax } from '@audius/common/utils'
+import type { CoinInsights } from '@audius/sdk'
+
+// TODO: Move all of this to a shared location with web
+
+export type MetricData = {
+  value: string
+  label: string
+  change?: {
+    value: string
+    isPositive: boolean
+  }
+}
+
+const messages = {
+  pricePerCoin: 'Price per coin',
+  holdersOnAudius: 'Holders on Audius',
+  uniqueHolders: 'Unique Holders',
+  volume24hr: 'Volume (24hr)',
+  totalTransfers: 'Total Transfers'
+}
+
+const CURRENCY_FORMAT_MAX = 100_000
+
+const formatPercentage = (num: number): string => {
+  return `${num >= 0 ? '+' : ''}${num.toFixed(2)}%`
```

Comment:
super nit: would be weird to see +0%
```suggestion
  return `${num > 0 ? '+' : ''}${num.toFixed(2)}%`
```